plugins/LadspaEffect/swh/analogue_osc_1416.c

```diff
@@ -175,7 +175,6 @@ static void runAnalogueOsc(LADSPA_Handle instance, unsigned long sample_count) {
 	float otm2 = plugin_data->otm2;
 	unsigned int rnda = plugin_data->rnda;
 	unsigned int rndb = plugin_data->rndb;
-	blo_h_tables * tables = plugin_data->tables;
```

Comment:
Once these are merged, we should issue a PR against https://github.com/swh/ladspa